pkg/upgrade/result/v1alpha1/upgraderesult_test.go

```diff
@@ -0,0 +1,72 @@
+package v1alpha1
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/upgrade/v1alpha1"
+)
+
+func TestWithAPIList(t *testing.T) {
+	tests := map[string]struct {
+		expectedURName []string
```

Comment:
A UT struct should be always designed with `inputs` & `expected outputs` in mind.
Can you redesign this accordingly?
It is okay to provide the same arguments twice one as input & other as expected output in UT.
In this case, we can even check the len of inputs as an expectation i.e. `expectedLen int`